src/site/content/en/authors/individual.njk

```diff
@@ -20,8 +20,36 @@ pagination:
 
   <div class="w-layout-container w-pb--non">
     <header class="w-page-header">
+      <img src="{{paged.data.hero}}" alt="{{paged.data.alt}}" class="w-author-page__image">
       <h1 class="w-page-header__headline">{{ renderData.title }}</h1>
       <p class="w-page-header__copy">{{ renderData.description | safe }}</p>
+      <div class="w-author-page__links">
+        <div>
```

Comment:
This extra div makes it so your flexbox CSS doesn't really do anything. I think it should be removed.